blend-library/src/main/java/com/wealthfront/blend/properties/AdditiveProperty.kt

```diff
@@ -104,3 +105,11 @@ interface AdditiveIntProperty<in Subject> : AdditiveProperty<Subject> {
       .toFloat()
   }
 }
+
+/**
+ * An [AdditiveProperty] specifically designed for views. Handles storing running animations for you.
+ */
+interface AdditiveViewProperty<in Subject : View> : AdditiveProperty<Subject> {
```

Comment:
Added this because it makes more sense than keeping this logic in the helper function